_templates/ripple/tide-module/index.js.ejs.t

```diff
@@ -1,17 +1,10 @@
 ---
-to: packages/<%= name %>/index.js
+to: packages/ripple-tide-<%= h.changeCase.paramCase(name) %>/index.js
 ---
 
-import { getLandingPageComponents } from '@dpc-sdp/ripple-tide-api/src/services/utils'
-import componentMapping from './component-mapping'
-import components from './component-loader'
```

Comment:
I stripped out all the extra stuff in here that was related to landing page so this is a lot more lightweight